### DISCUSSION TITLE
A cycle was detected when generating the classpath

### DISCUSSION THREAD

**Discussion by zhangxian314** [dbeaver/dbeaver#19947] [DBeaver Development]
### Description

when i export product:
'export product' has encountered a problem.
A cycle was detected when generating the classpath 
org.jkiss.dbeaver.ext.greenplum_1.0.105.202305091039, 
org.jkiss.dbeaver.ext.postgresql_2.1.177.202305091039, 
org.jkiss.dbeaver.ext.greenplum_1.0.105.202305091039.

### DBeaver Version

23.0.2

### Operating System

windows 10 64bit

**Comment by LonwoLonwo** [comment]
Hello @zhangxian314 

Please provide clear steps to reproduce your issue.
Could you please also provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Comment by zhangxian314** [comment]
1.open DBeaver.product, select Overview-->Export an eclipse product.
![image](https://github.com/dbeaver/dbeaver/assets/32084905/283f3874-67fa-42f9-8443-a2170c2660ab)
2.Configuration parameters
![image](https://github.com/dbeaver/dbeaver/assets/32084905/222bc329-4c02-4462-a02f-1d801741fbd0)
3.click finish.
4.'export product' has encountered a problem.
![image](https://github.com/dbeaver/dbeaver/assets/32084905/8b934579-10fb-4cf8-9b0c-5ff3de2a0d15)

But I can run DBeaver using the source code directly. 
such as:
![image](https://github.com/dbeaver/dbeaver/assets/32084905/87a5cc50-3d75-4e13-b353-1391b192cbf8)

**Comment by ShadelessFox** [comment]
Build the product using `mvn clean package`. See https://github.com/dbeaver/dbeaver/wiki/Build-from-sources
